packages/core-components/src/lib/unsorted/ui/Sidebar.svelte

```diff
@@ -191,13 +201,3 @@
 		</div>
 	{/if}
 </aside>
-
-<style>
-	.selected {
-		@apply text-blue-600;
-	}
-
-	.selected:hover {
-		@apply text-blue-600;
-	}
-</style>
```

Comment:
Original behaviour is that a selected sidebar link, when hovered, stays the selected colour. This makes a selected sidebar link have the same hover behaviour as an unselected one.